ARM-wvd-templates/DSC/Functions.ps1

```diff
@@ -0,0 +1,496 @@
+#Microsoft.RDInfra.RDPowerShell and Get-Package both require powershell 5.0 or higher.
+#Requires -Version 5.0
```

Comment:
I have been tolde before that Functions.ps1 exists in https://github.com/Azure/RDS-Templates/blob/master/wvd-templates/DSC/Functions.ps1 and should not have a copy under ARM-wvd-templates folder.

Reply:
Hmm should it be zipped into Configuration.zip?  When I tried testing it didnt work unless Configuration.zip had it

Reply:
`Functions.ps1` has to be inside `Configuration.zip` regardless whether we keep a copy of it inside the folder `Configuration`. Because the same exact file is used for classic WVD templates inside `wvd-templates/DSC`, we should not maintain its duplicate. Initially the plan was to keep a copy inside `wvd-templates/DSC` but not in side `ARM-wvd-templates/DSC` but as I think about this more, it might be a good idea to keep a copy here and not in classic given that there will more changes and deployments in ARM than in classic going forward. Any thoughts ? + @lintFan

Reply:
Just remove this file from this PR.

Reply:
Ok, I needed to know what is included in Configurations.zip though, since that is also in the PR.

Reply:
@nakranimohit0 I agree we should move the Functions.ps1 from `wvd-templates/DSC` to `ARM-wvd-templates/DSC`, I can do that outside of this PR so that it is easier see what changed and I'll make sure to maintain the history by doing a rename.